v2/pkg/model/types/stringslice/stringslice.go

```diff
@@ -67,6 +67,37 @@ func (stringSlice StringSlice) MarshalJSON() ([]byte, error) {
 	return json.Marshal(stringSlice.Value)
 }
 
+func (stringSlice *StringSlice) UnmarshalJSON(data []byte) error {
+	var marshalledValueAsString string
+	var marshalledValuesAsSlice []string
+
+	sliceMarshalError := json.Unmarshal(data, &marshalledValuesAsSlice)
+	if sliceMarshalError != nil {
+		stringMarshalError := json.Unmarshal(data, &marshalledValueAsString)
+		if stringMarshalError != nil {
+			return stringMarshalError
+		}
+	}
+
+	var result []string
+	switch {
+	case len(marshalledValuesAsSlice) > 0:
+		result = marshalledValuesAsSlice
+	case utils.IsNotBlank(marshalledValueAsString):
+		result = strings.Split(marshalledValueAsString, ",")
+	default:
+		result = []string{}
+	}
+
+	values := make([]string, 0, len(result))
+	//nolint:gosimple,nolintlint //cannot be replaced with result = append(result, slices...) because the values are being normalized
```

Comment:
Is it possible that the lint error was a false positive? I'm not getting any warning with the latest version of `golangci-lint`